README.md

```diff
@@ -34,14 +34,15 @@ Options:
   --overwrite          Overwrite any existing DMG.
   -h --help            Show this screen.
   --version            Show version.
-
 ```
 
 ### API
 
 ```javascript
-var createDMG = require('electron-installer-dmg')
-createDMG(opts, function done (err) { })
+const createDMG = require('electron-installer-dmg')
+createDMG(opts).then(() => {
+  // ...
+})
```

Comment:
suggestion(blocking): I'd prefer an example along the lines of either:

* https://electron.github.io/electron-packager/v15.2.0/modules/electronpackager.html
* https://github.com/electron/get#usage

I think showing the `async`/`await` syntax as an example is preferable to raw Promise syntax.

Reply:
I updated the readme accordingly.